openweathermap-ios/openweathermap-ios/BookmarkLocation/Manager/BookmarkLocationManager.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BookmarkLocationService.swift
+//  openweathermap-ios
+//
+//  Created by Cassio Sousa on 24/04/20.
+//
+
+import Foundation
+
+class BookmarkLocationManager: NSObject {
+
+    func save(bookmarkLocation: BookmarkLocation) -> [BookmarkLocation] {
```

Comment:
Why are you returning all here?

Reply:
I return because, I already have all data "in hands" then I prefer return all. In  backend I do this because usually have all data when save those and I don't need do another query in this case call the method getAll again. However if you want I can remove that.

Do you want that I remove that situation at method save?

Reply:
No, let it go. I just wanted to know the motivations. It's ok.